.tina/schema.ts

```diff
@@ -0,0 +1,241 @@
+                            },
+                            {
+                              name: 'value',
+                              type: 'reference',
+                              collection: 'doc',
+                              label: 'Value',
+                            },
+                          ],
+                        },
+                      ],
+                    },
+                  ],
+                },
+                {
+                  name: 'guideSection',
+                  label: 'Guide Section',
+                  fields: [
+                    {
+                      name: 'title',
+                      type: 'text',
+                      label: 'Title',
+                    },
+                    {
+                      name: 'subItems',
+                      type: 'group-list',
+                      label: 'Sub Items',
+                      fields: [
+                        {
+                          name: 'label',
+                          type: 'text',
+                          label: 'Label',
+                        },
+                        // {
+                        //   name: 'value',
+                        //   type: 'text',
+                        //   label: 'Value',
+                        // },
+                      ],
+                    },
+                  ],
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    },
+    {
+      label: 'Doc',
+      name: 'doc',
+      path: 'content/docs',
+      templates: [
+        {
+          name: 'doc',
+          label: 'Doc Teamplte',
+          fields: [
+            {
+              name: 'title',
+              type: 'text',
+              label: 'Title',
+            },
+            {
+              name: 'author',
+              type: 'text',
+              label: 'Author',
+            },
+            {
+              name: 'date',
+              type: 'text',
+              label: 'Date',
+            },
+            {
+              name: 'prev',
+              type: 'reference',
+              label: 'Prev',
+              collection: 'doc',
+            },
+            {
+              name: 'id',
+              type: 'text',
+              label: 'ID',
+            },
+            {
+              name: 'last_edited',
+              type: 'text',
+              label: 'Last Edited',
+            },
+            {
+              name: 'next',
+              type: 'reference',
+              label: 'Next',
+              collection: 'doc',
+            },
+            consumesField,
+          ],
+        },
+      ],
+    },
+    {
+      label: 'Blog',
+      name: 'blog',
+      path: 'content/blog',
+      templates: [
+        {
+          name: 'basic',
+          label: 'Basic',
+          fields: [
+            {
+              name: 'date',
+              type: 'text',
+              label: 'Date',
+            },
+            {
+              name: 'author',
```

Comment:
A blog post can have maany authors: 
e.g: https://tinacms-site-next-git-evolution-of-inline-editing-tinacms.vercel.app/blog/evolution-of-inline-editing/